_includes/head.html

```diff
@@ -71,15 +71,9 @@
       }
     }
   </style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
-</head>
-<head>
-  <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-
   <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
   <meta name="description" content="{{ page.excerpt | default: site.description | strip_html | normalize_whitespace | truncate: 160 | escape }}">
-  <link href="{{ site.github.url }}/assets/css/main.css" rel="stylesheet">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css, " | prepend: site.baseurl }}">
```

Comment:
for link use something like this:
`{{ "/xxx.jpg" | prepend: site.github.url | replace: 'http://', 'https://' }}`
using **baseurl** will cause issues as forks & main repos BaseURLs are different